lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -253,6 +253,7 @@ def delete_user(self, uname):
                     error=str(err)
                 )
                 return msg
+        [pref.delete() for pref in user.preferences.all()]
```

Comment:
It's better Python style to use a for statement: list comprehensions should only be used when you want the list of values it creates.

But shouldn't the Django ORM be deleting those preferences when we delete the user? We shouldn't have to delete them ourselves, I wouldn't have thought?